README.md

```diff
@@ -229,7 +229,7 @@ Resources:
 
 If you have any question, feel free to reach out to us in the following ways:
 
-[Volcano Slack Channel](https://cloud-native.slack.com/archives/C011GJDQS0N) | [Join](https://slack.cncf.io/)
+[Volcano Slack Channel](https://cloud-native.slack.com/archives/C011GJDQS0N) | [Join](https://join.slack.com/t/volcano-sh/shared_invite/enQtNTU5NTU3NDU0MTc4LTgzZTQ2MzViNTFmNDg1ZGUyMzcwNjgxZGQ1ZDdhOGE3Mzg1Y2NkZjk1MDJlZTZhZWU5MDg2MWJhMzI3Mjg3ZTk)
```

Comment:
Is this ckoud native computating foundation workspace?

Reply:
I am not sure https://app.slack.com/client/TGDR7QQF4/CGET876H5 is still alive?

Reply:
Yeah this is the right one.

Reply:
Seems this https://join.slack.com/t/volcano-sh/shared_invite/enQtNTU5NTU3NDU0MTc4LTgzZTQ2MzViNTFmNDg1ZGUyMzcwNjgxZGQ1ZDdhOGE3Mzg1Y2NkZjk1MDJlZTZhZWU5MDg2MWJhMzI3Mjg3ZTk is an old one
<img width="751" alt="image" src="https://github.com/user-attachments/assets/2e1e6719-55c2-419d-aa97-0f6f7f8c4ef9" />

Reply:
We should invite people to this channel https://cloud-native.slack.com/archives/C011GJDQS0N